src/zstd.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := zstd
+$(PKG)_WEBSITE  := https://github.com/facebook/$(PKG)
+$(PKG)_DESCR    := Zstandard is a fast lossless compression algorithm
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.3.2
+$(PKG)_CHECKSUM := ac5054a3c64e6510bc1ae890d05e3d271cc33ceebc9d06ac9f08105766d2798a
+$(PKG)_GH_CONF  := facebook/zstd,v
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_BUILD
+    # build and install the library
+    cd '$(BUILD_DIR)' && $(TARGET)-cmake \
+        -DCMAKE_INSTALL_BINDIR=$(BUILD_DIR)/null \
+        -DCMAKE_INSTALL_MANDIR=$(BUILD_DIR)/null \
+        '$(SOURCE_DIR)/build/cmake'
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
+    $(if $(BUILD_SHARED), $(INSTALL) '$(BUILD_DIR)/lib/libzstd.dll' '$(PREFIX)/$(TARGET)/bin/')
```

Comment:
The program, shared and static libs are installed for all targets, use the `ZSTD_BUILD_*` options:
```diff
 define $(PKG)_BUILD
     # build and install the library
-    cd '$(BUILD_DIR)' && $(TARGET)-cmake \
-        -DCMAKE_INSTALL_BINDIR=$(BUILD_DIR)/null \
-        -DCMAKE_INSTALL_MANDIR=$(BUILD_DIR)/null \
-        '$(SOURCE_DIR)/build/cmake'
+    cd '$(BUILD_DIR)' && $(TARGET)-cmake '$(SOURCE_DIR)/build/cmake' \
+        -DZSTD_BUILD_STATIC=$(CMAKE_STATIC_BOOL) \
+        -DZSTD_BUILD_SHARED=$(CMAKE_SHARED_BOOL) \
+        -DZSTD_BUILD_PROGRAMS=OFF
     $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
     $(MAKE) -C '$(BUILD_DIR)' -j 1 install
-    $(if $(BUILD_SHARED), $(INSTALL) '$(BUILD_DIR)/lib/libzstd.dll' '$(PREFIX)/$(TARGET)/bin/')
```